Mathlib/Data/Complex/Module.lean

```diff
@@ -485,10 +485,10 @@ lemma Real.rank_rat_real : Module.rank ℚ ℝ = continuum := by
 @[simp]
 lemma Complex.rank_rat_complex : Module.rank ℚ ℂ = continuum := by
   refine (Free.rank_eq_mk_of_infinite_lt ℚ ℂ ?_).trans mk_complex
```

Comment:
These proofs look like good arguments for reverting the arguments to explicit...

bors d+

Either way is fine by be